components/auth/auth.tsx

```diff
@@ -1,27 +1,46 @@
-import { Button, Image } from '@chakra-ui/react';
+import {
+  Button,
+  Image,
+  ButtonGroup
+} from '@chakra-ui/react';
+import { FaSpotify } from 'react-icons/fa';
 import { NextPage } from 'next';
 import { signIn } from 'next-auth/react';
 import styles from './auth.module.scss';
 
 const AuthLayout: NextPage = () => {
   return (
-    <div className={styles.card}>
-      <Image 
+    <div
+      className={styles.home}
+      style={{
+        backgroundColor: 'teal',
+        width: '100%',
+        height: '100vh',
+        objectFit:'cover',
+      }}
```

Comment:
Remove this from here, and add it to the scss file.

Reply:
Si los paso al file de scss no toma los cambios bro,

Reply:
Listo